CHANGELOG

```diff
@@ -1,5 +1,6 @@
 2.8.0.dev (compared to 2.7.X)
 -----------------------------
+- deprecate and warn about the --genscript option
```

Comment:
You should add something like "Thanks cpommranz for the PR" or something (or use your real name)